### PR TITLE
Rgo2018 without neck

### PR DESCRIPTION
**Pull request by rokusottervanger**
All RGO2018 changes, but without the neck dynamixel changes.

### CODE REVIEW

File: parameters/hardware/actuators/soem.ops [hunk 1]
```diff
@@ -112,8 +112,12 @@ connect (BASE_SLAVE_2+".encoder2","BASE_ReadEncoders.enc4_in", ConnPolicy() ); #
 ### READ TORSO ENCODERS ###
 var int ENCODERBITS_T 		= 65536;					# Max encoderbits (16 bits)
 var int ENCODERCOUNTS_T		= 500*4;						# Counts per cycle
-var double GEARBOX_LOWER 	= 2.0/5.0; 						# Motor to drive train
-var double GEARBOX_UPPER 	= 3.0/13.0; 					# Motor to drive train
+#var double GEARBOX_LOWER 	= 2.0/5.0; 						# Motor to drive train
+#var double GEARBOX_UPPER 	= 3.0/13.0; 					# Motor to drive train
+
+
+var double GEARBOX_LOWER 	= 1.0/8.75; 					# Motor to drive train
```
Comment (jlunenburg) [inline parameters/hardware/actuators/soem.ops:119]: * Remove old values
* What's the 'exact' value (typically not the rounded `1.0/x` but `number of teeth/number of teeth`

File: parameters/hardware/actuators/torso_joint_publisher.ops [hunk 2]
```diff
@@ -27,18 +27,21 @@ var string	ROBOT 			= os.getenv("ROBOT")
 
 # Motor, gearbox and encoder specs
 var double 	TWOPI 			= 2.0*3.141592
-var int ENCODERBITS 		= 65536;					# Max encoderbits (16 bits)
+var int ENCODERBITS 		= 65536;						# Max encoderbits (16 bits)
```
Comment (jlunenburg) [inline parameters/hardware/actuators/torso_joint_publisher.ops:30]: Remove old values

File: parameters/hardware/actuators/torso_joint_publisher.ops [hunk 3]
```diff
@@ -27,18 +27,21 @@ var string	ROBOT 			= os.getenv("ROBOT")
...
+var double METER2RAD		= 500.0*TWOPI;					# translation to rotation (rad/m)
 var array ENC2METER			= array(TWOPI/ENCODERCOUNTS*GEARBOX_LOWER/METER2RAD, 
-								TWOPI/ENCODERCOUNTS*GEARBOX_UPPER/METER2RAD);
+								    TWOPI/ENCODERCOUNTS*GEARBOX_UPPER/METER2RAD);
 var int NBITS 				= 8; 						# number of bits to the backhoff stack (Brake)
 var array SI2BITS 			= array(-1.0,-1.0);
 #var array VOLT2BITS 		= array(-5.0/1023.0, -5.0/1023.0) 
 var array VOLT2BITS 		= array(-1023.0/5.0, -1023.0/5.0) 
-var array OFFSET_CALIPHERS 	= array(0.25030, -0.33840);
+#var array OFFSET_CALIPHERS 	= array(0.25030, -0.33840);
```
Comment (jlunenburg) [inline parameters/hardware/actuators/torso_joint_publisher.ops:43]: Cleanup

File: parameters/hardware/sensors/back_laser_filters_parameters.yaml [hunk 4]
```diff
@@ -12,6 +12,16 @@ scan_filter_chain:
         type: filters/MultiChannelMedianFilterFloat
         params:
           number_of_observations: 3
+- type: laser_filters/LaserScanBoxFilter
```
Comment (jlunenburg) [inline parameters/hardware/sensors/back_laser_filters_parameters.yaml:15]: @reinzor @MatthijsBurgh @MaxBaeten do you know if this is correct? (git blame says Rokus but he has nothing to do with it)

File: parameters/hardware/sensors/base_laser_filters_parameters.yaml [hunk 5]
```diff
@@ -12,6 +12,16 @@ scan_filter_chain:
         type: filters/MultiChannelMedianFilterFloat
         params:
           number_of_observations: 3
+- type: laser_filters/LaserScanBoxFilter
+  name: laser_box_filter
```
Comment (jlunenburg) [inline parameters/hardware/sensors/base_laser_filters_parameters.yaml:16]: @reinzor @MatthijsBurgh @MaxBaeten do you know if this is correct? (git blame says Rokus but he has nothing to do with it)

File: parameters/hardware/sensors/torso_laser_filters_parameters.yaml [hunk 6]
```diff
@@ -12,6 +12,16 @@ scan_filter_chain:
         type: filters/MultiChannelMedianFilterFloat
         params:
           number_of_observations: 3
+- type: laser_filters/LaserScanBoxFilter
+  name: laser_box_filter
```
Comment (jlunenburg) [inline parameters/hardware/sensors/torso_laser_filters_parameters.yaml:16]: @reinzor @MatthijsBurgh @MaxBaeten do you know if this is correct? (git blame says Rokus but he has nothing to do with it)

File: parameters/world_modeling/face_recognition.yaml [hunk 7]
```diff
@@ -1 +1,2 @@
-camera_topic: /sergio/top_kinect/rgbd
+align_path: ~/openface/models/dlib/shape_predictor_68_face_landmarks.dat
```
Comment (jlunenburg) [inline parameters/world_modeling/face_recognition.yaml:1]: @Rayman is this correct?

File: parameters/world_modeling/world_model_spr.yaml [hunk 8]
```diff
@@ -87,27 +70,21 @@ plugins:
...
+  enabled: 1
   parameters:
       robot_name: sergio
       initial_pose_topic: /sergio/initialpose
       num_particles: 500
       initial_pose:
-          x: 0.586 #$(env ROBOT_INIT_X)
-          y: 4.259 #$(env ROBOT_INIT_Y)
-          rz: 0 #$(env ROBOT_INIT_PHI)
+          x: 1.5 #$(env ROBOT_INIT_X)
```
Comment (jlunenburg) [inline parameters/world_modeling/world_model_spr.yaml:85]: Make up-to-date with robotics-testlabs again??

**Review by jlunenburg — DISMISSED** [review]

**Comment by jlunenburg** [comment]
@MaxBaeten can you have a look at the .ops files as soon as your here? @Rayman I guess you updated most of the rest. It's a bit hard to review but can you take a look at my comments?

File: parameters/hardware/sensors/back_laser_filters_parameters.yaml (hunk 4 above)
Comment (MatthijsBurgh) [reply parameters/hardware/sensors/back_laser_filters_parameters.yaml:15]: Indeed, this is me. I have tuned and tested it on the robot. So as long as we want all laser points within the base to be removed, it is correct.

File: parameters/hardware/sensors/base_laser_filters_parameters.yaml (hunk 5 above)
Comment (MatthijsBurgh) [reply parameters/hardware/sensors/base_laser_filters_parameters.yaml:16]: Same

File: parameters/hardware/sensors/torso_laser_filters_parameters.yaml (hunk 6 above)
Comment (MatthijsBurgh) [reply parameters/hardware/sensors/torso_laser_filters_parameters.yaml:16]: Same

**Review by Rayman — APPROVED** [review]
Tournament mode on!

**Pull request merged**